### DISCUSSION TITLE
Dynamically adding label configs from neo4j server to neovis config

### DISCUSSION THREAD

**Discussion by Niveydhithaa** [neo4j-contrib/neovis.js#289] [Q&A]
Hi guys, I have defined a labels config for the existing nodes' labels like 
![image](https://user-images.githubusercontent.com/108443521/190408530-ae8c426b-599f-4802-9706-db9ba743d1dc.png)
. 

TO my requirement, say I have new nodes with a different with a new label in neo4j, then my neovis render shows up just the new nodes empty without its name. I have no idea how to do this. 

Can someone help me out?

**Comment by Niveydhithaa** [comment]
I wish neovis can extract all labels, and apply the same properties as given manually above. Is there any way?

**Comment by Cybernetic1** [comment]
Can you make a bigger screenshot, including the whole config structure?
